src/ledger/shred_inserter/merkle_root_checks.zig

```diff
@@ -300,15 +300,15 @@ fn previousErasureSet(
         defer for (entry) |e| e.deinit();
         const key = try key_serializer.deserialize(ErasureSetId, allocator, entry[0].data);
         if (key.slot != slot) return null;
-        if (key.fec_set_index != fec_set_index) break .{
+        if (key.erasure_set_index != erasure_set_index) break .{
             key,
             try value_serializer.deserialize(ErasureMeta, allocator, entry[1].data),
         };
     } else return null;
 
     // Check if this is actually the consecutive erasure set
     const next = if (candidate.nextFecSetIndex()) |n| n else return error.InvalidErasureConfig;
```

Comment:
would we want to change `nextFecSetIndex` too ?

Reply:
56202b7b2789bc9b525ff28159282530d4c430fb